app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,39 @@
+class RentalsController < ApplicationController
+
+  def check_out
+
+    rental = Rental.new(rental_params)
+
+    if rental.movie.available_to_rent?
+      if rental.save
+        rental.movie.reduce_available_inventory
+        puts rental.customer.movies_checked_out_count
+        rental.customer.increase_movies_checked_out_count
```

Comment:
Doing all this work should be in one method in the model. Maybe even something like `Movie#check_out(customer_id)`